tools/ops/script_runner/README.md

```diff
@@ -0,0 +1,115 @@
+# Set of Hue scripts to help manage hue.  While some of these are self contained, the most important
```

Comment:
Maybe reformat a bit the README to be more readable?

e.g. 

# Ops Script
Set of ....

    git clone ...

Reply:
And maybe renaming the commit `Add hue scripts from Chris Conner's personal repo` to something more generic
(I could squash all the commits when pushing too if sompler)

Reply:
Done.